server/services/personalPageService.ts

```diff
@@ -105,22 +97,35 @@ export default class PersonalPageService {
     return apiClient.getDistinguishingMarks(prisonerNumber)
   }
 
+  async getPhysicalAttributes(token: string, prisonerNumber: string): Promise<CorePersonPhysicalAttributes> {
+    const apiClient = this.personIntegrationApiClientBuilder(token)
+    return apiClient.getPhysicalAttributes(prisonerNumber)
+  }
+
   async updatePhysicalAttributes(
     token: string,
     user: PrisonUser,
     prisonerNumber: string,
-    physicalAttributes: Partial<PrisonPersonPhysicalAttributesUpdate>,
+    physicalAttributes: CorePersonPhysicalAttributesRequest,
   ) {
-    const apiClient = this.prisonPersonApiClientBuilder(token)
-    const response = await apiClient.updatePhysicalAttributes(prisonerNumber, physicalAttributes)
+    const apiClient = this.personIntegrationApiClientBuilder(token)
+
+    const existingPhysicalAttributes =
+      (await apiClient.getPhysicalAttributes(prisonerNumber)) ??
```

Comment:
By this point we've already retrieved the physical attributes once in the controller so we're ending up making the same call twice? Maybe it's fine, we're not going to have loads of these requests...